packages/playground/package.json

```diff
@@ -110,6 +110,7 @@
     "rimraf": "^3.0.2",
     "sass": "^1.54.5",
     "sass-loader": "^13.0.2",
+    "source-map-loader": "^4.0.0",
```

Comment:
If you rebase to `master` this and the `package-lock.json` changes will probably go away